eZ/Publish/Core/Repository/UserService.php

```diff
@@ -80,7 +80,7 @@ public function __construct(RepositoryInterface $repository, Handler $userHandle
             'defaultUserPlacement' => 12,
             'userClassID' => 4,// @todo Rename this settings to swap out "Class" for "Type"
             'userGroupClassID' => 3,
-            'hashType' => User::PASSWORD_HASH_MD5_USER,
+            'hashType' => User::PASSWORD_BCRYPT,
```

Comment:
On the other hand, `PASSWORD_BCRYPT` is also the constant used by PHP (ref: http://php.net/manual/en/function.password-hash.php ) so keeping it as-is is consistent with PHP.